src/v0/destinations/intercom/data/INTERCOMGroupConfig.json

```diff
@@ -0,0 +1,59 @@
+[
+  {
+    "destKey": "company_id",
+    "sourceKeys": "groupId",
+    "required": true
+  },
+  {
+    "destKey": "name",
+    "sourceKeys": ["traits.name","context.traits.name"],
+    "required": false
+  },
+  {
+    "destKey": "plan",
+    "sourceKeys": ["traits.plan","context.traits.plan"],
+    "required": false
+  },
+  {
+    "destKey": "size",
+    "sourceKeys": ["traits.size","context.traits.size"],
+    "metadata": {
+      "type": "toNumber"
+    },
+    "required": false
+  },
+  {
+    "destKey": "website",
+    "sourceKeys": ["traits.website","context.traits.website"],
+    "required": false
+  },
+  {
+    "destKey": "industry",
+    "sourceKeys": ["traits.industry","context.traits.industry"],
+    "required": false
+  },
+  {
+    "destKey": "monthly_spend",
+    "sourceKeys": ["traits.monthlySpend","context.traits.monthlySpend"],
+    "metadata": {
+      "type": "toNumber"
+    },
+    "required": false
+  },
+  {
+    "destKey": "remote_created_at",
+    "sourceKeys": ["traits.remoteCreatedAt","context.traits.remoteCreatedAt"],
+    "metadata": {
+      "type": "toNumber"
+    },
+    "required": false
+  },
+  {
+    "destKey": "custom_attributes",
+    "sourceKeys": ["traits.customAttributes","context.traits.customAttributes"],
```

Comment:
We shouldn't expect customAttributes like this. We should put all the traits not on our mapping to custom_attributes by default.

Reply:
updated